### DISCUSSION TITLE
Flexbox Froggy, a cute little game for learning CSS Flexbox

### DISCUSSION THREAD

**Discussion by rh5140** [albertkun/21S-ASIAAM-191A#60] [General]
Someone introduced [Flexbox Froggy](https://flexboxfroggy.com/) to me a while back, and I thought it was a fun way to learn a little CSS. Sharing in case anyone else wants to try it out!
![image](https://user-images.githubusercontent.com/40257341/114287897-c386ff00-9a1f-11eb-83c9-405d6907ddc3.png)

**Comment by albertkun** [comment]
Great resource Ray!! The Grid garden will be useful for CSS-Grid which is what we will mainly be using in class for UI/UX!

![image](https://user-images.githubusercontent.com/8574425/114334494-81cd8580-9aff-11eb-985f-b5f6c40a7f9d.png)

https://cssgridgarden.com/
